cloud/deployment/inspect/inspect.go

```diff
@@ -0,0 +1,157 @@
+package inspect
```

Comment:
nitpick: should `inspect` be an independent folder in itself? do we expect more commands/files to be added in the future?

Reply:
Good question and we are not necessarily going to add more commands or files to inspect but it is easier to read and maintain if it is in its own package/dir.

Reply:
Typically it is easier to read and maintain if it is in its own package. The pattern I am following is below: